### PR TITLE
notebook re-articulations of environmental triad google sheets consensus

### PR DESCRIPTION
**Pull request by turbomam**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -29,6 +31,8 @@ mkdocs-mermaid2-plugin = "^0.6.0"
 oaklib = "^0.5.28" # https://pypi.org/project/oaklib/0.5.6/
 rdflib = "^6.2.0"
 sheets-and-friends = "^0.5.4"
+jupyter = "^1.1.1"
```
Comment (sierra-moxon) [inline pyproject.toml:34]: I like these down in dev deps, is there any chance the ones above can go down here too?

**Comment by sierra-moxon** [comment]
non-blocking review comments:
- if we can make some subdirectories in the notebooks directory to separate gold from env_triad terms that would be 👍 
- if we can put all new python packages into the dev dependencies that would be 👍 too
- a small README for order of operations would be 👍

**Review by sierra-moxon — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (turbomam) [reply pyproject.toml:34]: yes! my slip-up.

**Comment by turbomam** [comment]
@sierra-moxon I think I addressed all of your non-blocking suggestions. Could you please review again? If you approve, I suggest mergoing this into 

- https://github.com/microbiomedata/submission-schema/pull/260

And then merging that

**Review by sierra-moxon — APPROVED** [review]

**Pull request merged**